server/src/db/migrations/20231101183711_updateUsers.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.alterTable("users", (table) => {
+        table.string("image")
```

Comment:
now with AWS set up, you could manually add an image to the S3 bucket, and use its URL to set a default image with `defaultTo` if a user has not provided an image